src/packages/@ncigdc/components/Aggregations/RangeFacet.js

```diff
@@ -261,91 +337,99 @@ const RangeFacet = (props: TProps) => {
-                />
+                  type="radio"
+                  value="years"
+                  />
                 Years
               </label>
               <label
                 htmlFor={`${dotField}-days-radio`}
                 style={{ paddingRight: '10px' }}
-              >
+                >
                 <input
-                  type="radio"
-                  value="days"
-                  onChange={props.handleUnitChanged}
                   checked={props.state.selectedUnit === 'days'}
                   id={`${dotField}-days-radio`}
+                  onChange={props.handleUnitChanged}
                   style={{ marginRight: '5px' }}
-                />
+                  type="radio"
+                  value="days"
+                  />
                 Days
               </label>
             </form>
           </Row>
-        )}
+      )}
       {!props.collapsed && (
         <Column>
           <Row>
             <InputLabel
+              htmlFor={`from-${dotField}`}
               style={{
                 borderRight: 0,
                 borderRadius: '4px 0 0 4px',
               }}
-              htmlFor={`from-${dotField}`}
-            >
+              >
               From:
             </InputLabel>
             <Input
-              value={props.state.fromDisplayed || ''}
-              onChange={props.handleFromChanged}
               id={`from-${dotField}`}
               key={`from-${dotField}`}
-              type="number"
-              placeholder={`eg. ${minDisplayed}`}
               max={maxDisplayed}
               min={minDisplayed}
-              title="todo"
-            />
+              onChange={props.handleFromChanged}
+              placeholder={`eg. ${minDisplayed}`}
+              style={{
+                paddingRight: '5px',
+                paddingLeft: '5px',
+              }}
+              title={`eg. ${minDisplayed}`}
+              type="number"
+              value={props.state.fromDisplayed || ''}
+              />
             <InputLabel
+              htmlFor={`to-${dotField}`}
               style={{
                 borderLeft: 0,
                 borderRight: 0,
               }}
-              htmlFor={`to-${dotField}`}
-            >
+              >
               To:
             </InputLabel>
             <Input
-              value={props.state.toDisplayed || ''}
-              onChange={props.handleToChanged}
               id={`to-${dotField}`}
               key={`to-${dotField}`}
-              type="number"
-              placeholder={`eg. ${maxDisplayed}`}
               max={maxDisplayed}
               min={minDisplayed}
-              title="todo"
-            />
+              onChange={props.handleToChanged}
+              placeholder={`eg. ${maxDisplayed}`}
+              style={{
+                paddingRight: '5px',
+                paddingLeft: '5px',
+              }}
+              title={`eg. ${maxDisplayed}`}
```

Comment:
change title to `Max value: eg. ${maxDisplayed}`

Reply:
Done.